src/modules/launcher/Wox.Test/QueryBuilderTest.cs

```diff
@@ -194,8 +197,9 @@ public void QueryBuilder_ShouldSetTermsCorrently_WhenCalled()
             var secondQuery = pluginQueryPairs.GetValueOrDefault(secondPlugin);
 
             // Assert
-            Assert.IsTrue(firstQuery.Terms[0].Equals("cd") && firstQuery.Terms[1].Equals("efgh") && firstQuery.Terms.Length == 2);
-            Assert.IsTrue(secondQuery.Terms[0].Equals("efgh") && secondQuery.Terms.Length == 1);
+            // Using InvariantCulture since this is a command line arg
+            Assert.IsTrue(firstQuery.Terms[0].Equals("cd", StringComparison.InvariantCulture) && firstQuery.Terms[1].Equals("efgh", StringComparison.InvariantCulture) && firstQuery.Terms.Length == 2);
```

Comment:
I just want to make sure the use of InvariantCulture is intentional here.  This will treat different characters with the same linguistic meaning 'æ' and 'ae' as equal.    I don't think it makes a difference here but we should be intentional with the parameters we are setting.

Reply:
Thanks @ryanbodrug-microsoft. I've updated the PR to use Ordinal in place of InvariantCulture.